From e0c7574ef2ea5a6a3cc8ff92bd03d72d3f73e739 Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Fri, 07 Mar 2014 23:15:06 +0000
Subject: Fix load from text file dialogue response

Avoid an assert by fixing the accepted response types to include those
given at construction.

https://retrace.fedoraproject.org/faf/problems/1581387/
---
diff --git a/src/misc.c b/src/misc.c
index af68db0..721b081 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -2724,7 +2724,7 @@ void Open_Load_Filename_Window (void)
                                                       GTK_STOCK_CLOSE,
                                                       GTK_RESPONSE_CANCEL,
                                                       GTK_STOCK_APPLY,
-                                                      GTK_RESPONSE_ACCEPT,
+                                                      GTK_RESPONSE_APPLY,
                                                       NULL);
     gtk_dialog_set_default_response (GTK_DIALOG (LoadFilenameWindow),
                                      GTK_RESPONSE_ACCEPT);
--
cgit v0.9.2
